squirrellypenguin/Frozzy-React-Frontend
# Fe layout styling home

### What does this PR do?
- Create and added styling to Main page for review
- Home page styling is currently sitting outside of the HTML body for testing purposes. 

### Is this a feature or a fix?
- Feature

## If it's a fix, please describe. 
- NA

### Where should the reviewer start?
- index.js (Lines 45-96)
- index.css (lines 17-110)

### How should this be tested?
- Open mobile view and look to check if formatting matches comp
- NOTE: Desktop view not formatted 

### Issues: 
- Background vector image (ice cream background) may have to change scale after 

### NOTES: 
- We'll have to do a little tweaking to the Classes & ID's when formatting into the MAIN component
- Button is non functional
